components/p3a/brave_p3a_service.cc

```diff
@@ -392,7 +392,7 @@ void BraveP3AService::OnHistogramChanged(base::StringPiece histogram_name,
     base::PostTask(FROM_HERE, {content::BrowserThread::UI},
                    base::BindOnce(&BraveP3AService::OnHistogramChangedOnUI,
                                   this,
-                                  histogram_name,
+                                  base::StringPiece(histogram_name),
```

Comment:
I would change `OnHistogramChangedOnUI` signature as well

Reply:
Fixed in 099563129b726f01d59ba5c4203af15a2e43ed23